src/styles/components/header.scss

```diff
@@ -1,16 +1,16 @@
 .header {
-  flex: 0 1;
-  display: flex;
-  background-color: $mainDark-color;
   position: relative;
-  justify-content: flex-end;
+  display: flex;
+  flex: 0 1;
   align-items: center;
+  justify-content: flex-end;
   padding: 20px $defaultSidePadding;
+  background-color: $mainDark-color;
```

Comment:
I usually use this website https://www.color-blindness.com/color-name-hue/ when I need to assign names to colors. It rescues me in such time